### PR TITLE
Refactor mpybit: Fixed code quality, remove pylint disables and commented code

### PR DESCRIPTION
**Pull request by shekhar-cis**
Refactor mpybit: use `os.path.join`, remove pylint disables

### CODE REVIEW

File: src/bitmessagekivy/mpybit.py [hunk 1]
```diff
@@ -277,7 +282,7 @@ def exit_manager(self, *args):
     def load_selected_Image(self, curerentAddr):
         """This method load the selected image on screen"""
         top_box_obj = self.root.ids.content_drawer.ids.top_box.children[0]
-        top_box_obj.source = self.image_dir + '/default_identicon/{0}.png'.format(curerentAddr)
+        top_box_obj.source = os.path.join(self.image_dir, '/default_identicon/{0}.png'.format(curerentAddr))
```
Comment (PeterSurda) [inline src/bitmessagekivy/mpybit.py:285]: missing `join`

File: src/bitmessagekivy/screens_data.json [hunk 2]
```diff
@@ -1,25 +1,11 @@
...
     "Payment": {
         "kv_string": "payment",
         "name_screen": "payment",
-        "Import": "from pybitmessage.bitmessagekivy.baseclass.payment import Payment"
-    },
-    "MailDetail": {
-        "kv_string": "maildetail",
-        "name_screen": "mailDetail",
-        "Import": "from pybitmessage.bitmessagekivy.baseclass.maildetail import MailDetail"
+        "Import": "from bitmessagekivy.baseclass.payment import Payment"
```
Comment (PeterSurda) [inline src/bitmessagekivy/screens_data.json:5]: add `pybitmessage.`

**Review by PeterSurda — CHANGES_REQUESTED** [review]

File: src/bitmessagekivy/screens_data.json (hunk 2 above)
Comment (shekhar-cis) [reply src/bitmessagekivy/screens_data.json]: Done

File: src/bitmessagekivy/mpybit.py (hunk 1 above)
Comment (shekhar-cis) [reply src/bitmessagekivy/mpybit.py]: Done

File: setup.py [hunk 3]
```diff
@@ -79,7 +79,8 @@ def run(self):
         packages.extend(
             [
                 'pybitmessage.bitmessagekivy',
-                'pybitmessage.bitmessagekivy.baseclass'
+                'pybitmessage.bitmessagekivy.baseclass',
+                'pybitmessage.bitmessagekivy.tests'
```
[the remark this answers is not in the document]
Comment (shekhar-cis) [reply setup.py]: Done

**Review by PeterSurda — APPROVED** [review]

File: src/bitmessagekivy/mpybit.py [hunk 4]
```diff
@@ -86,12 +88,14 @@ class NavigateApp(MDApp):
     identity_list = get_identity_list()
     image_path = load_image_path()
     app_platform = platform
+    Window.size = (500, 800)
```
[the remark this answers is not in the document]
Comment (shekhar-cis) [reply src/bitmessagekivy/mpybit.py]: Done

File: src/bitmessagekivy/mpybit.py [hunk 5]
```diff
@@ -247,11 +255,11 @@ def file_manager_open(self):
...
         """This method is used to set the select image"""
         try:
-            from PIL import Image as PilImage
             newImg = PilImage.open(path).resize((300, 300))
             if platform == 'android':
                 android_path = os.path.join(
-                    os.environ['ANDROID_PRIVATE'] + '/app' + '/images' + '/kivy/')
+                    os.path.join(os.environ['ANDROID_PRIVATE'], 'app', 'images', 'kivy/')
+                )
                 if not os.path.exists(android_path + '/default_identicon/'):
```
Comment (PeterSurda) [inline src/bitmessagekivy/mpybit.py:263]: `os.path.join`

File: src/bitmessagekivy/mpybit.py [hunk 6]
```diff
@@ -247,11 +255,11 @@ def file_manager_open(self):
     def select_path(self, path):
         """This method is used to set the select image"""
         try:
-            from PIL import Image as PilImage
             newImg = PilImage.open(path).resize((300, 300))
             if platform == 'android':
                 android_path = os.path.join(
-                    os.environ['ANDROID_PRIVATE'] + '/app' + '/images' + '/kivy/')
+                    os.path.join(os.environ['ANDROID_PRIVATE'], 'app', 'images', 'kivy/')
```
Comment (PeterSurda) [inline src/bitmessagekivy/mpybit.py:261]: I don't like the `/` at the end

**Review by PeterSurda — CHANGES_REQUESTED** [review]

**Review by PeterSurda — CHANGES_REQUESTED** [review]
- unified resolution
- fix `format`

**Review by PeterSurda — APPROVED** [review]
Squash and if all tests pass, I can merge.

**Review by PeterSurda — APPROVED** [review]

**Pull request merged**